plugins/module_utils/kibana.py

```diff
@@ -207,6 +207,7 @@ def format_alert_params(self):
       formatted_params['filterQuery'] = self.module.params.get('filter_query')
     if self.module.params.get('group_by'):
       formatted_params['groupBy'] = self.module.params.get('group_by')
+      formatted_params['alertOnGroupDisappear'] = self.module.params.get('alert_on_group_disappear')
```

Comment:
Do we expect `alert_on_group_disappear` to always be provided to the module when `group_by` exists? If not, I would argue for a nested if statement here to check for `alert_on_group_disappear` so we aren't including null values in the parameters.

Reply:
Added check for `None` and supplied a default value of `False`. The default when this value isn't specified is `True`, which is almost never desirable, as groups tend to change in normal operation and the resulting notifications are worded in a confusing way. (Basically, something like Low Disk Space would trigger if a previously mounted drive was unmounted, and would send a notification with the wording "Low Disk Space")